google/cloud/spanner/internal/spanner_stub.cc

```diff
@@ -235,14 +238,46 @@ StatusOr<spanner_proto::PartitionResponse> DefaultSpannerStub::PartitionRead(
   return response;
 }
 
+bool RpcTracingEnabled() {
+  auto enable_clog =
+      google::cloud::internal::GetEnv("GOOGLE_CLOUD_CPP_ENABLE_CLOG");
+  if (enable_clog.has_value()) {
+    (void)google::cloud::LogSink::EnableStdClog();
```

Comment:
Curious: does the compiler or clang-tidy or something require this cast to void?

Reply:
No, that was a mistake. Fixed now.